### ISSUE TITLE
Data Export Issues

### ISSUE THREAD

**Issue by loriabendroth** [isudatateam/datateam#375]
@akrherz , few things to address:

- [ ] rows 2 and 3 (description and units) are blank in all of my spreadsheet tabs
- [ ] need to export the items listed under management and metadata tab (such as dictionary, etc). Now, only the plot data/measurements are exported.
- [ ] widen columns in export. Right now they are each ~1 inch wide. Make all of them ~1.5 inch wide; should be plenty of room for that.
- [ ] remove index column from all sheets.

**Comment by loriabendroth** [comment]
@akrherz , should I test the interface after these commits you made to see if better? Or are you still working?

**Comment by akrherz** [comment]
> should I test the interface after these commits you made to see if better? Or are you still working?

No, I don't think it is worth it yet.  I got things more working than they were before, there's some stuff to do yet.

**Comment by loriabendroth** [comment]
@akrherz , the index numbers (column A in each tab) are still present. Can you remove? Also, columns are a bit too wide now. Reduce width by 25%.

**Comment by akrherz** [comment]
> the index numbers (column A in each tab) are still present. Can you remove? Also, columns are a bit too wide now. Reduce width by 25%

Both should now be completed.  The index numbers sneaking back in was a bit perplexing.
